tools/rnc-prettyprint.sh

```diff
@@ -0,0 +1,7 @@
+# This assumes you have trang installed via package manager
+# which installs a script to provide a "trang" command.
+# Make sure you run this before commiting any changes to the 
+# rnc files.
```

Comment:
```
# Make sure you run this before commiting any changes to the 
# rnc files.
```

It would be nice if CI could take of this. Same for https://github.com/citation-style-language/styles/, where I currently run https://formatter.citationstyles.org/ manually on most pull requests, with periodic runs of https://github.com/citation-style-language/utilities/blob/master/csl-reindenting-and-info-reordering.py to catch any pretty printing we missed during merging.

It looks like Github Actions can be used for this, but only for PRs originating from the same repo (see https://github.com/stefanzweifel/git-auto-commit-action#unable-to-commit-into-prs-from-forks).

Reply:
Isn't the problem with this that, by definition, CI actions take place after commit?

That's why I was wondering about a pre-commit hook.

Reply:
Yes. (for the styles repo, pre-commit hooks don't make much sense, since we have so many infrequent/one-off contributors, and a lot of contributions that come in via the GitHub website instead of a local client, but I guess you could use them here)